src/Model/Package.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Describes a installed composer package version.
+ *
+ * This is a DataObject to cache the a package's information for later use.
+ */
+class Package extends DataObject
+{
+    private static $db = [
+        'Name' => 'Varchar(255)',
+        'Description' => 'Varchar(255)',
+        'Version' => 'Varchar(255)',
+        // 'Direct' => 'Boolean',
+    ];
```

Comment:
Is this the replacement for ComposerPackageVersion from the silverstripe-composer-versions module? E.g. does this change fix https://github.com/bringyourownideas/silverstripe-maintenance/issues/46 ?